package.json

```diff
@@ -11,44 +11,44 @@
     "lint": "eslint assets"
   },
   "dependencies": {
-    "autoprefixer": "^6.5.1",
-    "babel-cli": "^6.18.0",
-    "babel-core": "^6.18.0",
-    "babel-loader": "^6.2.7",
-    "babel-preset-es2015": "^6.18.0",
-    "babel-preset-react": "^6.16.0",
-    "bootstrap": "4.0.0-alpha.4",
-    "bootstrap-loader": "^1.2.1",
+    "autoprefixer": "^6.7.2",
+    "babel-cli": "^6.22.2",
+    "babel-core": "^6.22.1",
+    "babel-loader": "^6.2.10",
+    "babel-preset-es2015": "^6.22.0",
+    "babel-preset-react": "^6.22.0",
+    "bootstrap": "^4.0.0-alpha.4",
```

Comment:
While Bootstrap  v4 doesn't reach a beta version, we can't rely on `^`. Lock the version by removing this character.